cmd/ctr-remote/commands/convert.go

```diff
@@ -69,6 +73,19 @@ When '--all-platforms' is given all images in a manifest list must be available.
 			Usage: "eStargz chunk size",
 			Value: 0,
 		},
+		cli.IntFlag{
+			Name:  "estargz-min-chunk-size",
+			Usage: "The minimal number of bytes of data must be written in one gzip stream. Note that this adds a TOC property that old reader doesn't understand.",
+			Value: 0,
+		},
+		cli.BoolFlag{
+			Name:  "estargz-external-toc",
+			Usage: "Separate TOC JSON into another image (called \"TOC image\"). The name of TOC image is the original + \"-esgztoc\" suffix. Both eStargz and the TOC image should be pushed to the same registry. stargz-snapshotter refers to the TOC image when it pulls the result eStargz image.",
+		},
+		cli.BoolFlag{
+			Name:  "estargz-lossless",
```

Comment:
"Lossless images" may sound like graphic images as in PNG images 😆 

`estargz-keep-diff-id` might be less confusing, but no strong opinion

Reply:
Thank you for the suggestion. Renamed to flag to `estargz-keep-diff-id`.